### PR TITLE
[FSTORE-612] Add docs for feature monitoring

### PR DESCRIPTION
**Pull request by javierdlrm**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -72,11 +74,16 @@ nav:
               - user_guides/fs/feature_group/index.md
               - Create: user_guides/fs/feature_group/create.md
               - Create External: user_guides/fs/feature_group/create_external.md
-              - Deprecating: user_guides/fs/feature_group/deprecation.md
+              - Deprecate: user_guides/fs/feature_group/deprecation.md
```
Comment (robzor92) [inline mkdocs.yml:77]: Maybe `Deprecated` instead?

**Review by robzor92 — CHANGES_REQUESTED** [review]

File: docs/css/custom.css [hunk 2]
```diff
@@ -1,58 +1,67 @@
 :root {
```
[the remark this answers is not in the document]
Comment (javierdlrm) [reply docs/css/custom.css:1]: I wanted to use admonitions with title but without icon, but that's not an option provided by the tool we use. But they have support for **custom icons**. This is a custom icon which is basically hidden.

File: docs/user_guides/fs/feature_group/data_validation_best_practices.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+First you will need to configure your preferred communication endpoint: slack, email or pagerduty. Check out [this page](../../../admin/alert.md) for more information on how to set it up. A typical use-case would be to add an alert on ingestion success to a Feature Group you created to hold data that failed validation. Here is a quick walkthrough:
+
+1. Go the Feature Group page in the UI
+2. Scroll down and click on the `Add an alert` button.
+3. Choose the trigger, receiver and severity and click save.
+
+## Conclusion
+
+Hopsworks completes Great Expectation by automatically running the validation, persisting the reports along your data and allowing you to monitor data quality in its UI. How you decide to make use of these tools depends on your application and requirements. Whether in development or in production, real-time or batch, we think there is configuration that will work for your team. Check out our [quick hands-on tutorial](https://colab.research.google.com/github/logicalclocks/hopsworks-tutorials/blob/master/integrations/great_expectations/fraud_batch_data_validation.ipynb) to start applying what you learned so far.
```
[the remark this answers is not in the document]
Comment (javierdlrm) [reply docs/user_guides/fs/feature_group/data_validation_best_practices.md:112]: This page is already released. I've only refactored it by extracting the "best practices" and "advanced guide" into two separate pages to make them easier to find.

File: docs/user_guides/fs/feature_group/feature_monitoring.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+Before continuing with this guide, see the [Feature monitoring guide](../feature_monitoring/index.md) to learn more about how feature monitoring works, and get familiar with the different use cases of feature monitoring for Feature Groups described in the **Use cases** sections of the [Scheduled statistics guide](../feature_monitoring/scheduled_statistics.md#use-cases) and [Statistics comparison guide](../feature_monitoring/statistics_comparison.md#use-cases).
+
+!!! warning "Limited UI support"
+    Currently, feature monitoring can only be configured using the **Hopsworks Python library**. However, you can enable/disable a feature monitoring configuration or trigger the statistics comparison manually from the UI, as shown in the [Advanced guide](../feature_monitoring/feature_monitoring_advanced.md).
+
+## Code
+
+In this section, we show you how to setup feature monitoring in a Feature Group using the ==Hopsworks Python library==. Alternatively, you can get started quickly by running our [tutorial for feature monitoring](https://github.com/logicalclocks/hopsworks-tutorials/blob/master/integrations/feature-monitoring/feature-monitoring.ipynb).
```
[the remark this answers is not in the document]
Comment (javierdlrm) [reply docs/user_guides/fs/feature_group/feature_monitoring.md:12]: I will work on the tutorial after finishing the documentation. It's not merged yet, but that's the expected url.

File: docs/user_guides/fs/feature_group/data_validation_best_practices.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (javierdlrm) [reply docs/user_guides/fs/feature_group/data_validation_best_practices.md:112]: Same as previous answer. But I will fix it anyways.

File: docs/user_guides/fs/feature_group/feature_monitoring.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (javierdlrm) [reply docs/user_guides/fs/feature_group/feature_monitoring.md:12]: Same as previous answer. I'll work on the tutorial after finishing the docs. That's the expected url for the tutorial

File: docs/user_guides/fs/feature_group/data_validation_best_practices.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (javierdlrm) [reply docs/user_guides/fs/feature_group/data_validation_best_practices.md:112]: Fixed.

[the remark this answers is not in the document]
Comment (javierdlrm) [reply docs/user_guides/fs/feature_group/data_validation_best_practices.md:112]: Fixed

File: docs/user_guides/fs/feature_group/feature_monitoring.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
+# Feature Monitoring for Feature Groups
+
+Feature Monitoring complements the Hopsworks data validation capabilities for Feature Groups by allowing you to monitor your data once they have been ingested into the Feature Store. Hopsworks feature monitoring is centered around two functionalities: **scheduled statistics** and **statistics comparison**.
+
+Before continuing with this guide, see the [Feature monitoring guide](../feature_monitoring/index.md) to learn more about how feature monitoring works, and get familiar with the different use cases of feature monitoring for Feature Groups described in the **Use cases** sections of the [Scheduled statistics guide](../feature_monitoring/scheduled_statistics.md#use-cases) and [Statistics comparison guide](../feature_monitoring/statistics_comparison.md#use-cases).
+
+!!! warning "Limited UI support"
+    Currently, feature monitoring can only be configured using the **Hopsworks Python library**. However, you can enable/disable a feature monitoring configuration or trigger the statistics comparison manually from the UI, as shown in the [Advanced guide](../feature_monitoring/feature_monitoring_advanced.md).
```
[the remark this answers is not in the document]
Comment (javierdlrm) [reply docs/user_guides/fs/feature_group/feature_monitoring.md:8]: Fixed

File: docs/user_guides/fs/feature_view/feature_monitoring.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+
+Before continuing with this guide, see the [Feature monitoring guide](../feature_monitoring/index.md) to learn more about how feature monitoring works, and get familiar with the different use cases of feature monitoring for Feature Views described in the **Use cases** sections of the [Scheduled statistics guide](../feature_monitoring/scheduled_statistics.md#use-cases) and [Statistics comparison guide](../feature_monitoring/statistics_comparison.md#use-cases).
+
+!!! warning "Limited UI support"
+    Currently, feature monitoring can only be configured using the **Hopsworks Python library**. However, you can enable/disable a feature monitoring configuration or trigger the statistics comparison manually from the UI, as shown in the [Advanced guide](../feature_monitoring/feature_monitoring_advanced.md).
+
+## Code
+
+In this section, we show you how to setup feature monitoring in a Feature View using the ==Hopsworks Python library==. Alternatively, you can get started quickly by running our [tutorial for feature monitoring](https://github.com/logicalclocks/hopsworks-tutorials/blob/master/integrations/feature-monitoring/feature-monitoring.ipynb).
```
[the remark this answers is not in the document]
Comment (javierdlrm) [reply docs/user_guides/fs/feature_view/feature_monitoring.md:12]: Fixed.

File: mkdocs.yml (hunk 1 above)
Comment (javierdlrm) [reply mkdocs.yml:77]: I would leave it as `Deprecate`, similar to `Create`.
It is a guide on how to deprecate a Feature Group.

**Review by robzor92 — APPROVED** [review]

**Pull request merged**